.idea/Sprint-Challenge--User-Interface.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Consider adding a `.gitignore` file to your project. Then add the `.idea` folder so that these files are not uploaded to Github.

Look at this link and add `.idea` to your list: https://gist.github.com/octocat/9257657